_SETUP_DEV_ENVIRONMENT.md

```diff
@@ -22,6 +22,64 @@ There are several ways to set up your development server and environment. Basica
 
 Note: to regenerate the docker configuration, use `php artisan sail:install --devcontainer`
 
+### Windows (natively)
+On Windows, probably the optimal option with respect to performance is
+to simply install a web server and a database server natively on the system,
```

Comment:
This is not natively, it's based on Xampp, which is a developer tool and also does a lot of "magic" in the background. I had issues with Xampp before, which did not support the newest PHP versions. Apparently, I see that currently this is not the case, however, we should make a note here to check it before even trying to set up.